src/e2e-tests/page-objects/smartthings/outsideTab.page.js

```diff
@@ -0,0 +1,40 @@
+// outsideTab.page.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+
+class OutsideTabPage extends Page {
```

Comment:
shall not extend page because it is not a page, it is more a component, set of options